### PR TITLE
Feature 1 development done

### PR DESCRIPTION
**Pull request by sviswanath57**
we created as part of development

- modified the code
- modified the line

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -1,2 +1,7 @@
 ###
-This is the first commit from main branch
\ No newline at end of file
+This is the first commit from main branch
+
+geature-1 commit-1
+feature-1 commit-2
+geature-1 commit-3
```
Comment (sviswanath57) [inline README.MD:6]: you can update the user name with parameters

**Review by sviswanath1914 — COMMENTED** [review]
Is this line really required ?

**Review by sviswanath57 — COMMENTED** [review]
yes its required

**Review by sviswanath1914 — APPROVED** [review]
approved

**Pull request merged**